src/error.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _ERROR_H
+#define _ERROR_H
+
+void kernel_panic(const char *msg);
```

Comment:
Please write a comment about what this does and how it should be used.

Reply:
done!